clean_ceph_vols.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+from gevent import monkey, sleep
+
+monkey.patch_all()
+import yaml
+import sys
+import os
+import logging
+from docopt import docopt
+from libcloud.common.types import LibcloudError
+from ceph.utils import get_openstack_driver, volume_cleanup
+from ceph.parallel import parallel
+from utility.retry import retry
+from utility.utils import (
+    timestamp,
+    create_run_dir,
+)
+
+
+doc = """
+A simple test suite wrapper that executes tests based on yaml test configuration
```

Comment:
Could we re-write the document string so that we can bring out the intent i.e. perform cleanup of orphaned volumes.

Reply:
Updated docstring